airbyte-integrations/connectors/source-stripe/source_stripe/error_handlers/parent_incremental_stripe_sub_stream_error_handler.py

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+#
+
+from typing import Optional, Union
+
+import requests
+from airbyte_cdk.sources.streams.http.error_handlers.response_models import ErrorResolution
+from source_stripe.error_handlers.stripe_error_handler import StripeErrorHandler
+
+
+class ParentIncrementalStripeSubStreamErrorHandler(StripeErrorHandler):
+    def interpret_response(self, response_or_exception: Optional[Union[requests.Response, Exception]] = None) -> ErrorResolution:
+        if response_or_exception.status_code == requests.codes.not_found:
```

Comment:
This code will fail if `response_or_exception` is of type `Exception` hence we should handle this case before calling `response_or_exception.status_code`

Reply:
Right, let me update this, thanks.

Reply:
Fixed